sections/04-lensing.md

```diff
@@ -1,6 +1,16 @@
+
+## Lensing as Method
+
+As a practice-based research method, lensing starts from Gillian Rose's assertion that arts-based research functions not within an empirical, positivist epistemology where "to know is to see," but in an interpretive mode where, "for studio-researchers, to _know is to see ... differently_" (quoted in Sullivan 85).
+
+Though different, lensing shares characteristics with Mieke Bal's concept of _framing_. In her book, _Travelling Concepts in the Humanities_, Bal emphasizes that framing is an active process of contextualization, interpretation, and analysis (135-6). In defiance of the noun form ("a frame"), framing's methodological action--with its emphasis on temporality, duration, and change--aims to call into question the boundaries and status of its object of study. "As a verb, [framing]... also predicates that object, not in the abstract void of theoretical reflection, but in time, space, aspect; it frames it. Thus, all by itself, even on the level of the word alone, 'framing' questions the object-status of the objects studied in the cultural disciplines. This questioning results in a repositioning of the object as alive" (137).
+
+While lensing shares with framing the processual and temporal qualities of what I called earlier a "theory in motion," lensing is primarily experiential rather than interpretive in character. It is a method that aims to find ways to alter and transform of our modes of seeing, listening, and thinking via an engagement with what Karen Barad calls the "material-discursive phenomena" that constitute bodies in the world--a complex system or apparatus of concepts _and_ materials comprising both the human and non-human (Barad 822).
+
+I have generalized this concept of lensing from my software architectural research at the Inclusive Design Research Centre (where it was originally borrowed from Benjamin Pierce's work on programming languages for data transformation) (Pierce x). There, lensing is a specific technical strategy that involves the creation of networks of "functional-reactive" relays that adapt and transform software state in response to change within the system. Lenses are implemented in a manner that enables them to be altered or replaced at any time, including in the midst of actually using the software. The goal of lensing, as it is implemented in the _Fluid Infusion_ programming framework<sup>2</sup> is to provide a "living" means to connect together software bodies from different individuals and communities without requiring their interface to be homogeneous<sup>3</sup>. Lensing, from the technological perspective, supports the movement of Fluid's "open community" model from simply a method (as it is commonly practiced in open source software development communities) into a characteristic of _the artifact of software itself_.
```

Comment:
Actually I'm not sure that liveness with respect to the lens structure itself is necessarily a concomitant of Pierce's view of what lenses are, even though it is of ours... I've only see him be interested in liveness with respect to the endpoints of the lens.